src/patternfly/components/Tile/tile.scss

```diff
@@ -24,20 +27,18 @@
   --pf-c-tile__icon--Color: var(--pf-global--Color--200);
   --pf-c-tile--hover__icon--Color: var(--pf-global--primary-color--100);
   --pf-c-tile--m-selected__icon--Color: var(--pf-global--primary-color--100);
-  --pf-c-tile--m-disabled__icon--Color: var(--pf-global--disabled-color--100);
```

Comment:
We should leave this in. If someone sets `--pf-c-tile__icon--Color: red;`, then it will still be red if it's disabled since we only set the icon color for stacked icons.

Reply:
Also need to add back so the stacked icon color is disabled

`--pf-c-tile__header--m-stacked__icon--Color: var(--pf-global--disabled-color--100);`